main.py

```diff
@@ -46,7 +47,7 @@ def request_github_internship24_data() -> List[Opportunity]:
 
     for cell in td_elems:
         temp.append(cell)
-        if len(github_list) < 10:
+        if len(github_list) < 15:
```

Comment:
The integer literal `15` repeats[^1]. Shall we assign that value to a constant variable?

[^1]: https://en.m.wikipedia.org/wiki/Don%27t_repeat_yourself

Reply:
Resolved [here](https://github.com/acmcsufoss/acmOpportunities/pull/19/commits/0a25691c94fa80f551275e70dec5c612b0014058)